docker/key-manager/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM debian:buster AS toolchain
```

Comment:
@mgorven, I think we should have a release docker container with all binaries. Then we can amend it with the docker-run.sh as appropriate. What are your thoughts?

Reply:
I think having one image with both safety-rules and key-manager makes sense. I'm not sure about including validator though, from a security perspective.

Reply:
@davidiw @mgorven is this something you'd like me to address in this PR, or submit as a separate PR?

Reply:
If we're going to do that (which makes sense to me) it's worth doing now. There's a bunch of other stuff to do for adding a new image which can be avoided that way (creating an ECR repo, updating CI to build it, etc.)

Reply:
Okay, awesome! Thanks @mgorven, I'll make that change today and update this PR when it's ready 😄